app/assets/stylesheets/application.scss

```diff
@@ -24,6 +38,10 @@ main {
   flex: 1 0 auto;
 }
 
+nav {
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.